File: app/src/main/java/com/sgr/owltube_v2/infra/webapi/response/popular/Thumbnails.java
package com.sgr.owltube_v2.infra.webapi.response.popular;

public class Thumbnails {

    public Default _default;
    public Medium medium;
    public High high;

    /**
     * No args constructor for use in serialization
     * 
     */
    public Thumbnails() {
    }

    /**
     * 
     * @param _default
     * @param high
     * @param medium
     */
    public Thumbnails(Default _default, Medium medium, High high) {
        this._default = _default;
        this.medium = medium;
        this.high = high;
    }

}

File: app/src/main/java/com/sgr/owltube_v2/infra/webapi/response/popular/Thumbnails.kt
package com.sgr.owltube_v2.infra.webapi.response.popular

import com.sgr.owltube_v2.infra.webapi.response.channels.Default
import com.sgr.owltube_v2.infra.webapi.response.channels.High
import com.sgr.owltube_v2.infra.webapi.response.channels.Medium

data class Thumbnails(val default: Default, val medium: Medium, val high: High)
